cpp/include/cudf/detail/aggregation/aggregation.cuh

```diff
@@ -18,20 +18,29 @@
 
 #include <cudf/aggregation.hpp>
 #include <cudf/column/column_device_view.cuh>
+#include <cudf/column/column_view.hpp>
 #include <cudf/detail/aggregation/aggregation.hpp>
 #include <cudf/detail/utilities/assert.cuh>
 #include <cudf/detail/utilities/device_atomics.cuh>
-#include <cudf/dictionary/dictionary_column_view.hpp>
-#include <cudf/table/table_device_view.cuh>
+#include <cudf/table/table_view.hpp>
 #include <cudf/utilities/traits.cuh>
 
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/exec_policy.hpp>
 
 #include <thrust/fill.h>
 
+#include <type_traits>
+#include <vector>
```

Comment:
Do we need `#include <vector>` since we moved a bunch of code?

Reply:
Just double-checked and they are indeed necessary.